unidist/test/test_general.py

```diff
@@ -45,6 +46,7 @@ def test_get_ip():
 
 
 def test_num_cpus():
-    import multiprocessing as mp
-
-    assert_equal(unidist.num_cpus(), mp.cpu_count())
+    if Backend.get() == "Python":
```

Comment:
Need to additional tests for `backend` and try more values

Reply:
Will be tested in CI when we add tests there.